doc/dsl/README.md

```diff
@@ -0,0 +1,92 @@
+
+**Taskorganisation**
+
+Eine Taskorganisation...
+- modelliert die Organisation mehrerer Aufgaben untereinander (Sequenz, Parallelismus, Verschachtelung)
+- orientiert sich konzeptionell an Petri-Netzen
+- legt Bedingungen für die Freischaltung von Aufgaben (Aufgabe kann bearbeitet werden) und den
+    Abschluss von Aufgaben fest (falls eine Aufgabe Unteraufgaben hat, gilt sie erst
+    als abgeschlossen, falls eine bestimmte Anzahl an Unteraufgaben abgeschlossen ist)
+
+**Level/Quest-Config**
+
+Eine Level/Quest-Config (der Name ist noch nicht final festgelegt):
+- beschreibt, welche Aufgaben in einem Level des Dungeons geladen werden
+- beschreibt, welche Szenarien für die Umsetzung der Aufgaben im Level verwendet
+    werden (vgl. [Szenarien](doc/tasks/README.md#Szenarien) // Note: Link ist von [PR #444](https://github.com/Programmiermethoden/Dungeon/pull/444) abhängig
+
+
+**Entitätstyp-Definition**
+
+Eine Entitätstyp-Definition:
+- legt fest, welche [Komponenten](../doc/ecs/components/readme.md) in einem Entitätstyp enthalten sind
+- bietet die Möglichkeit, die Werte der Komponenten zu definieren
+- kann weitergehen konfiguriert werden, d.h. die Werte der Komponenten einer bestehende Entitätstyp-Definition verändert und überschrieben werden, sodass eine konkrete Ausprägung eines Entitätstyps entsteht
+
+**Event-Handler-Funktion**
+
+Event-Handler-Funktionen bieten die Möglichkeit:
+- auf eine bestimmtes Ereignis aus dem Kontext des Dungeon-Levels oder einer Komponente zu reagieren (bspw.: "eine andere Hitbox kollidiert mit der Hitbox des `CollisionComponent` dieser Entität")
+- Parameter anzunehmen, welche den Kontext des Ereignisses beschreiben
+- im Rumpf Logik zu implementieren, welche native Funktionen aufrufen, bedingte Programmverzweigungen realisieren und auf Objekte des Dungeons zugreifen kann, um bestimmtes Verhalten zu erzeugen (bspw.: "Wenn die Entität der Hitbox mit der ich kollidiere ein Item ist, dann hebe ich das Item auf")
+
+**Taskbuilder-Methode**
+
+Die Taskbuilder-Methoden sind dafür zuständig, eine abstrakte Aufgabendefinition in ein konkretes Szenario im Dungeon Level zu übersetzen. Sie bekommen als Parameter die Taskdefinition übergeben und müssen eine Menge Entitäten zurückgeben, welche an dem Szenario beteiligt sind.
+
+### Quickstart Guide
+
+Der [Quickstart Guide](quickstart.md) gibt Hinweise für die Erstellung der ersten Aufgaben mit der DungeonDSL.
+
+### Weitere Dokumentation
+
+Die folgenden Dokumentationsseiten beleuchten einzelne Aspekte der Nutzung der DungeonDSL detaillierter als der Quickstart Guide.
+
+- Die [Sprachkonzept Dokumentation](sprachkonzepte.md) beschreibt die Sprachkonzepte der DungeonDSL auf einer tiefergehenden Ebene als der Quickstart Guide.
+- [Typsystem](typsystem.md) beschreibt, wie Typen in der DungeonDSL funktionieren.
+
+## DSL Pipeline
```

Comment:
Ist das ein Aspekt der "Weiteren Dokumentation"? Braucht es eine extra Überschrift oder reicht ein Link in das entsprechende Dokument?

Imho wäre es hier im Readme eine gute Idee, unter "Weitere Dokumentation" einfach auf die entsprechenden Aspekte zu verlinken und nicht noch tiefer in Details (und Gliederung) zu gehen.

Reply:
Das ist tatsächlich eher ein Aspekt der Entwicklerdokumentation, da das ja schon auf die Implementierung abzielt. Meine Idee war, dass man in dem README.md File oben alle userfacing Aspekte hat und es dann nach unten hin immer mehr (aber immer noch im Überblick) auf die Implementierung zu geht.

Reply:
Idealerweise würde sich dann das Bild, was unter "DSL Pipeline" landet, auf die [Grafik um den Zusammenhang aller Projekt-Bausteine zu visualisieren](https://github.com/Programmiermethoden/Dungeon/issues/501) beziehen.

Reply:
> Das ist tatsächlich eher ein Aspekt der Entwicklerdokumentation, da das ja schon auf die Implementierung abzielt. Meine Idee war, dass man in dem README.md File oben alle userfacing Aspekte hat und es dann nach unten hin immer mehr (aber immer noch im Überblick) auf die Implementierung zu geht.

Die Aufteilung ist sinnvoll, aber das Readme ist nur ein Überblick. Ich würde hier einfach nur _eine_ Überschrift "Weitere Dokumentation" (oder "Technische Dokumentation"?) einbauen und darunter dann passend benannte Links in die entsprechenden vertiefenden Dokumentationen setzen. Aber ich würde hier nicht wegen einem Link oder zwei Sätzen noch eine Überschrift einbauen.

Reply:
@cagix ist das hier resolved? Ich habe die Struktur ja angepasst.